cluster-up/scale.sh

```diff
@@ -0,0 +1,144 @@
+  while ! scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no /etc/yum.repos.d/$REPO_FILE root@$IP:/etc/yum.repos.d; do
+     sleep 5
+  done
+fi
+
+# Enable nested virtualization
+run_ssh rmmod kvm-intel
+run_ssh "echo 'options kvm-intel nested=1' >> /etc/modprobe.d/dist.conf"
+run_ssh modprobe kvm-intel
+
+# Add vagrant public key
+run_ssh "echo 'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eWW6A8YVr+kz4TjGYe7gHzIw+niNltGEFHzD8+v1I2YJ6oXevct1YeS0o9HZyN1Q9qgCgzUFtdOKLv6IedplqoPkcmF0aYet2PkEDo3MlTBckFXPITAMzF8dJSIFo9D8HfdOV0IAdx4O7PtixWKn5y2hMNG0zQPyUecp4pzC6kivAIhyfHilFR61RGL+GPXQ2MWZWFYbAGjyiYJnAmCP3NOTd0jMZEnDkbUvxhMmBYSdETk1rRgm+R4LOzFUGaHqHDLKLX+FIPKcF96hrucXzcWyLbIbEgE98OHlnVYCzRdK8jlqm8tehUc9c9WhQ== vagrant insecure public key' >> /root/.ssh/authorized_keys"
+
+# Add core user
+run_ssh useradd core
+run_ssh "echo core | passwd --stdin core"
+run_ssh mkdir -p /home/core/.ssh
+run_ssh cp .ssh/authorized_keys /home/core/.ssh
+run_ssh chown -R core /home/core/.ssh
+
+cp $KUBECONFIG /tmp
+sed -i 's/127.0.0.1/192.168.126.11/' /tmp/kubeconfig
+
+cat <<EOF > /usr/share/ansible/openshift-ansible/inventory/hosts
+[all:vars]
+ansible_user=root
+openshift_kubeconfig_path="/tmp/kubeconfig"
+
+[new_workers]
+$IP
+EOF
+
+
+cd /usr/share/ansible/openshift-ansible
+if [ $PROVISION_MODE = "1" ]; then
+  match=' - name: Apply ignition manifest'
+  insert=' - name: Goodbye\n    fail:\n      msg: All good\n    when: true\n'
+  file='./roles/openshift_node/tasks/config.yml'
+  sed -i.bak "s/$match/$insert\n $match/" $file
+
+  ansible-playbook -i inventory/hosts playbooks/scaleup.yml
+else
+  if [ $CUSTOM_IMAGE != "0" ]; then
+    match='- name: Install openshift support packages'
+    insert='  tags:\n    - skipme'
+    file='./roles/openshift_node/tasks/install.yml'
+    sed -i.bak "s/$match/$match\n$insert/" $file
+
+    match='  - name: Install openshift packages'
+    insert='    tags:\n      - skipme'
+    file='./roles/openshift_node/tasks/install.yml'
+    sed -i "s/$match/$match\n$insert/" $file
+
+    match=' - name: Pull release image'
+    insert='    tags:\n      - skipme'
+    file='./roles/openshift_node/tasks/config.yml'
+    sed -i.bak "s/$match/$match\n$insert/" $file
+
+    match='  - name: Pull MCD image'
+    insert='    tags:\n      - skipme'
+    file='./roles/openshift_node/tasks/config.yml'
+    sed -i "s/$match/$match\n$insert/" $file
+  fi
+
+  ansible-playbook -i inventory/hosts playbooks/scaleup.yml --skip-tags=skipme
+fi
+
+if [ $PROVISION_MODE = "1" ]; then
+  DOMAIN_NAME=$(virsh net-list | grep -v def | grep active | awk '{print $1}')-worker-1
+
+  function get_state {
+    STATE=$(virsh dominfo $DOMAIN_NAME | grep State | awk '{print $2}')
+  }
+
+  run_ssh "rm -f /etc/yum.repos.d/$REPO_FILE"
+  virsh shutdown $DOMAIN_NAME
+
+  while true
+  do
+     get_state
+     [ $STATE != "shut" ] || break
+     sleep 5
+  done
+
+  mv /tmp/el77-worker-1.img /tmp/el77-worker-ocp.img
+fi
```

Comment:
do we need a seperate `if` for this , or this can be onserted into this `if` : https://github.com/kubevirt/kubevirtci/pull/212/files#diff-18e81d0671724ea26dcf46c3a229f272R91-R97

-

Reply:
its possible indeed, just matter of taste if we want to separate the ansible run from the 
epilogue